client/src/components/loginComponents/StyledModal.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Check with @xuxiaqing2011 but I think you should be able to `import StyleModal from '../ModalTemplate/StyleModal.jsx` and then call the modal in the return statement and pass the children that you want it to render as props:

`<StyleModal children={what you want it to render} />`

This might prevent duplicates of the same component being created

Reply:
At this point, I've relabeled both of the files in the ModalTemplate folder. They should no longer be linked to anything, but can serve as modal templates to duplicate/edit where needed.

Reply:
I think we don't need a page for Login if I recall correctly.  You may just want to delete that. @andronicus217